src/BenchmarkDotNet/Helpers/ArtifactFileNameHelper.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using BenchmarkDotNet.Diagnosers;
+using BenchmarkDotNet.Exporters;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Toolchains;
+
+namespace BenchmarkDotNet.Helpers 
+{
+    internal static class ArtifactFileNameHelper
+    {
+        public static string GetFilePath(DiagnoserActionParameters details, DateTime creationTime, string fileExtension)
```

Comment:
good idea to move it to a separate file!

```suggestion
        internal static string GetFilePath(DiagnoserActionParameters details, DateTime creationTime, string fileExtension)
```